style.css

```diff
@@ -810,7 +764,7 @@ ul {
   width: 327px;
   height: 170px;
   border-radius: 8px;
-  margin-top: 10px;
+  padding: 10px;
 }
 
 .submission {
```

Comment:
Great job team 🤝

- [ ] Kindly fix the overlapping submit button on your mobile design as it deviates from the design provided in the Figma template.
- Your design
![overlap mobile](https://user-images.githubusercontent.com/63917725/180440688-675faef2-82b6-48fd-8bc4-bf17d158ee44.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/180440904-faebe583-3512-4227-868d-ce5f33485fd3.PNG)